src/invidious/jsonfilter.cr

```diff
@@ -0,0 +1,251 @@
+require "json"
```

Comment:
I'd recommend moving this into `src/invidious/helpers/handlers.cr` since it's middleware.

Reply:
I just removed it completely since it seems like`json` is already available in `handlers.cr` anyway.
It works well, but let me know if you'd prefer it be added back in somewhere.

Reply:
Sorry to be misleading. I mean to say that the entire `module JSONFilter` could be moved into `handlers.cr` or as a new file under `src/invidious/helpers/json_filter.cr` for the reason I mentioned.